spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require 'support/database_cleaner.rb'
+require 'devise'
```

Comment:
...unless you're requiring to gain access to the TestHelpers, but even that I think you can require directly.